packages/nouns-webapp/src/components/Proposals/Proposals.module.css

```diff
@@ -11,22 +11,50 @@
 }
 
 .heading {
-  display: inline-block;
-  font-size: 2rem;
+  font-family: 'Londrina Solid';
+  font-size: 40px;
   margin: 0 !important;
 }
 
-.createProposalLink {
-  padding: 0.9rem 1.5rem;
-  border-radius: 5px;
-  border: 0 none;
-  font-size: large;
-  font-weight: normal;
-  cursor: default;
+.generateBtn {
+  max-width: 10rem;
+  height: 3rem;
+  font-weight: bold;
+  margin-bottom: 0.5rem;
+  border-radius: 12px;
+  background-color: var(--brand-dark-red);
+  border: var(--brand-dark-red);
+}
+.generateBtn:focus,
+.generateBtn:hover {
+  background-color: var(--brand-dark-red) !important;
```

Comment:
What do you think about using green instead of dark red for "submit proposal" button? Feels like [red signals something bad](https://user-images.githubusercontent.com/85328329/154355443-77b5c10a-b672-451b-9aec-7873ad22c2a3.png). (something that needs to be done in playground as well).

Reply:
done! makes sense